mmdet3d/core/visualizer/show_result.py

```diff
@@ -75,7 +75,7 @@ def show_result(points,
                 pred_bboxes,
                 out_dir,
                 filename,
-                show=False,
```

Comment:
The --show in testing only controls the offline visualization and creates files which can be visualized by MeshLab. But the show in visualizer controls the online visualization. So you can add arguments in testing to control whether to visualize online.

Reply:
However, this actually made `--show` in `./tools/test.py` broke:
In test.py, it calls `single_gpu_test` with `args.show` as the argument, and when that is `True`, it will [call](https://github.com/open-mmlab/mmdetection3d/blob/b255d16b9f8a19a8f1065a1ae6c88fd2a03dc325/mmdet3d/apis/test.py#L37-L41) `model.module.show_results(...)`, where in the function it [calls](https://github.com/open-mmlab/mmdetection3d/blob/b255d16b9f8a19a8f1065a1ae6c88fd2a03dc325/mmdet3d/models/detectors/mvx_two_stage.py#L502) `mmdet3d.core.show_result` without `show` argument (which in all of previous versions, it is default to `True`).
In the newest version, the default value of `show` in `mmdet3d.core.show_result` is set to `False`, which makes all of the previous code broke, if we update to the new version (and this is not mentioned in the Changelog, and if other users have used it in different places assuming the original behavior of `show_result`, it will also broke).
Therefore, this default value should not be changed to `show=False`, and should remain `show=True` as previous versions.

Reply:
Thanks for your advice. Next, we will incorporate the 3D Segmentation into the repo. So we want to split the online visualization and offline visualization separately. For now, we can set the default `show` to true in show_results, which may be modified because of the consistency to 3D Segmentation.

Reply:
Thanks! And at that time, maybe we should change the behavior of `model.module.show_results(...)` by adding the parameter `show=True` when calling `mmdet3d.core.show_result`.

Reply:
You are kindly right about the behavior. And we will take the feature into account soon.